components/TextFields/src/MDCTextInputControllerOutlinedTextArea.h

```diff
@@ -32,8 +32,39 @@
  floating.
 
  This design defaults to 5 lines minimum and does not expand on overflow by default.
+
+ Defaults:
+
+ Active Color - Blue A700
+
+ Border Stroke Color - Clear
+ Border Fill Color - Clear
+
+ Disabled Color = [UIColor lightGrayColor]
+
+ Error Color - Red A400
+
+ Floating Placeholder Color Active - Blue A700
+ Floating Placeholder Color Normal - Black, 54% opacity
+
+ Inline Placeholder Color - Black, 54% opacity
+
+ Leading Underline Label Text Color - Black, 54% opacity
+
+ Normal Color - Black, 54% opacity
+
+ Rounded Corners - All
```

Comment:
Nit: Add radius value?

Reply:
Radius value is not currently parameterized. Created issue #2631